python/tchannel/req_handler.py

```diff
@@ -0,0 +1,69 @@
+import collections
+
+EndPoint = collections.namedtuple('EndPoint', ['handler', 'opts'])
+
+
+class RequestHandler(object):
+
+    def __init__(self):
+        pass
+
+    def handle_request(self, context, conn):
+        raise NotImplementedError()
```

Comment:
Document this class. Especially things like,

- How implementations are expected to behave.
- What the arguments for `handle_request` are.